README.md

```diff
@@ -2,8 +2,8 @@
 <h1 align="center"><img alt="cloudquery logo" width=75% src="https://github.com/cloudquery/cloudquery/raw/main/cli/docs/images/logo.png"/></h1>
 <!-- markdownlint-enable MD033 -->
 
-CloudQuery is an open-source,
-high-performance data integration platform for security and infrastructure teams.
+CloudQuery is an [open-source](https://github.com/cloudquery/cloudquery),
+high-performance data integration framework built for developers.
```

Comment:
Is it `framework` or `platform` now? The website has `platform` 😄 Personally I like `framework`

Reply:
yeah I think it is a bit interchangeable, we can lean more toward framework as we don't want to maintain the longtail of plugins.